elodie/tests/elodie_test.py

```diff
@@ -15,7 +15,7 @@
 sys.path.insert(0, os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))))))
 
 import helper
-elodie = load_source('elodie', os.path.abspath('{}/../../elodie.py'.format(os.path.dirname(os.path.realpath(__file__)))))
+elodie = load_source('elodie', os.path.abspath('{}/../elodie.py'.format(os.path.dirname(os.path.realpath(__file__)))))
```

Comment:
Is changing this path what's causing the tests to fail in Python 2.7?
https://travis-ci.org/jmathai/elodie/jobs/491249606

Reply:
I'm not exactly sure why the build is failing in 2.7, it doesn't seem to be an import error though. I haven't had time to investigate the 2.7 failure yet